docker/.env.example

```
@@ -15,3 +15,5 @@ ALLOWED_HOSTS=localhost,127.0.0.1
 DJANGO_SUPERUSER_USERNAME=admin
 DJANGO_SUPERUSER_EMAIL=admin@example.com
 DJANGO_SUPERUSER_PASSWORD=change-this-password
+
+TMDB_API_KEY=your-tmdb-api-key
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)